modules/op-release-notes-1-5.adoc

```diff
@@ -0,0 +1,299 @@
+====
+
+* The `ServiceType` and `podTemplate` fields are removed from the `EventListener` spec.
+
+* The controller service account no longer requests cluster-wide permission to list and watch namespaces.
+
+* The status of the `EventListener` resource has a new condition called `Ready`.
++
+[NOTE]
+====
+In the future, the other status conditions for the `EventListener` resource will be deprecated in favor of the `Ready` status condition.
+====
+
+* The `eventListener` and `namespace` fields in the `EventListener` response are deprecated. Use the `eventListenerUID` field instead.
+
+* The `replicas` field is deprecated from the `EventListener` spec. Instead, the `spec.replicas` field has been moved to `spec.resources.kubernetesResource.replicas` in the `KubernetesResource` spec.
++
+[NOTE]
+====
+The `replicas` field will be removed in a future release.
+====
+
+* The old method of configuring the core interceptors is deprecated. However, it will continue to work until it is removed in a future release. Instead, interceptors in a `Trigger` resource are now configured using a new `ref` and `params` based syntax. The resulting default webhook will automatically switch the usages of the old syntax to the new syntax for new triggers.
+
+* Use `rbac.authorization.k8s.io/v1` instead of the deprecated `rbac.authorization.k8s.io/v1beta1` for the `ClusterRoleBinding` resource.
+
+* In cluster roles, the cluster-wide write access to resources such as `serviceaccounts`, `secrets`, `configmaps`, and `limitranges` are removed. In addition, cluster-wide access to resources such as `deployments`, `statefulsets`, and `deployment/finalizers` are removed.
+
+* The `image` CRD in the `caching.internal.knative.dev group` is not used by Tekton anymore, and is excluded in this release.
+
+
+[id="known-issues-1-5_{context}"]
+== Known issues
+
+* The link:https://github.com/tektoncd/catalog/tree/main/task/git-cli/0.1[git-cli] cluster task is built off the link:https://github.com/tektoncd/catalog/blob/68e44c629c9ee287393681030ed391d2c2e856cd/task/git-cli/0.1/git-cli.yaml#L32[alpine/git] base image, which expects `/root` as the user's home directory. However, this is not explicitly set in the `git-cli` cluster task.
++
+In Tekton, the default home directory is overwritten with `/tekton/home` for every step of a task, unless otherwise specified. This overwriting of the `$HOME` environment variable of the base image causes the `git-cli` cluster task to fail.
++
+This issue is expected to be fixed in the upcoming releases. For {pipelines-title} 1.5 and earlier versions, you can _use any one of the following workarounds_ to avoid the failure of the `git-cli` cluster task:
+
+** Set the `$HOME` environment variable in the steps, so that it is not overwritten.
+
```

Comment:
```
Note: If you've installed OpenShift Pipelines via the operator, then the best way to do this is to clone to `git-cli` cluster task into a separate task and add the `HOME` environment variable in the step. While this is a bit cumbersome, this will make sure that the operator does not overwrite the changes that you've made to the cluster task.
```